projectile.el

```diff
@@ -3674,6 +3749,25 @@ to a custom function, else return nil."
         #'projectile--test-name-for-impl-name)
        (projectile-project-root)))))
 
+(defmacro projectile--acond (&rest clauses)
```

Comment:
I've added `acond` from `anaphora.el` (though with a tweak for emacs 25 compat) in an effort to make `projectile--find-matching-file` and `projectile--find-matching-test` a bit nicer.  If you prefer it I can remove it and go back to using the chain of `if-let`s.